sky/packages/sky/lib/src/rendering/proxy_box.dart

```diff
@@ -380,6 +429,9 @@ class RenderColorFilter extends RenderProxyBox {
   }
 }
 
+/// Clips its child using a rectangle
+///
+/// Prevents its child from painting outside its bounds.
```

Comment:
Is this cheaper than Opacity and IntrinsicWidth? You say those are expensive, but not this one.